datadog_checks_downloader/tests/conftest.py

```diff
@@ -58,6 +58,11 @@ def distribution_version(request):
     return request.config.getoption("--distribution-version")
 
 
+def pytest_generate_tests(metafunc):
+    if "disable_verification" in metafunc.fixturenames:
+        metafunc.parametrize("disable_verification", [False, True])
```

Comment:
Yes, all of this makes sense at a cursory glance. Is there an automated way to tell whether or not TUF and in-toto were used in one case or the other?

Reply:
Yes, PyTest includes the value of the "disable verification" fixture in the name of the test when running it. That's the "True" in

```
tests/test_downloader.py::test_local_download[True-datadog-active-directory-1.10.0-simple/datadog-active-directory/datadog_active_directory-1.10.0-py2.py3-none-any.whl] PASSED
```

See https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=132158&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=5a885e1d-de5f-5894-a427-14bbc366f16e&l=295

Reply:
No, I mean something deeper: other than analysing the code, is it possible to test whether or TUF/in-toto were actually hit/run in one branch or the other? Maybe you look at the log output at the very least, for example.

Reply:
OK, this sounds like call assertions.

Done in 92dd0e3